index.js

```diff
@@ -0,0 +1,54 @@
+let counter = 0;
+
+const carousel = document.getElementById('carousel')
+const imageContainer = document.querySelector('.carousel')
+
+const imageArr = ['https://placeimg.com/1280/720/nature', 'https://placeimg.com/1280/720/animals', 'https://placeimg.com/1280/720/tech']
+const heroPhrase = ['Good things', 'Welcome!', 'Wonderful day']
```

Comment:
semicolons are missing at the end of lines. You could use Prettier or similar configuration